test/resources/v3/program-orders.test.js

```diff
@@ -2,6 +2,11 @@ const nock = require('nock');
 const RO = require('../../..');
 const fixtures = require('../../fixtures/v3/program-orders.fixtures');
 
+jest.mock('../../../lib/utils/logger', () => ({
+  log: () => {},
+  prettyPrint: () => {},
+}));
```

Comment:
Jest should auto-mock the module methods, so we can simply go:
```suggestion
jest.mock('../../../lib/utils/logger');
```

Should we also add a few spy assertions to a few module tests, to ensure logs are called as expected?